hardware/eVehicles/MercApi.cpp

```diff
@@ -89,17 +89,40 @@ CMercApi::CMercApi(const std::string username, const std::string password, const
 
 CMercApi::~CMercApi()
 {
+	m_fields = "";
```

Comment:
I don't think this is needed in the destructor of a class

Reply:
removed